client/styles/style.css

```diff
@@ -345,21 +345,88 @@ select {
   font-family: 'Quicksand', 'Open Sans', sans-serif;
   font-size: 2em;
   margin-bottom: 20px;
+  display: inline-block;
 }
 
 .table-view {
   position: absolute;
-  max-height: 50%;
-  max-width: 100%;
+  max-height: 42.5%;
+  max-width: 80%;
   overflow-x: auto;
   overflow-y: auto;
 }
 
+/* CSV Table */
+.schema-csv-table {
+  font-size: 0.66em;
+  line-height: 40px;
+}
+
+td {
+  border: none;
+  overflow-x: auto;
+  padding: 5px 20px 5px 20px;
+}
+
 .csv-header {
-  cursor: pointer;
-  font-size: 0.75em;
+  background: teal;
+  color: #f2f2f2;
+  font-family: 'Quicksand', 'Open Sans', sans-serif;
+  font-weight: 400;
+  line-height: 20px;
+  padding: 5px 20px 5px 20px;
+  height: 50px;
 }
 
 .csv-header:hover {
   color: orange;
+  cursor: pointer;
+}
+
+.odd-header {
+  background: #007878;
+}
+
+.even-row .even-column {
+  background: #dae0e0;
+}
+
+.odd-row .even-column {
+  background: #e5e7e7;
+}
+
+.even-row .odd-column {
+  background: #cfd8d9;
+}
+
+.odd-row .odd-column {
+  background: #dae0e0;
+}
+
+.download-button,
+.upload-button {
+  font-size: 0.66em;
+  float: right;
+  border: 1px solid teal;
+  padding: 8px 15px 8px 15px;
+  border-radius: 3px;
+}
+
+.upload-button {
+  position: relative;
+  overflow: hidden;
+  margin: 0 0 0 5px;
+  color: teal;
+}
+
+.upload {
+  position: absolute;
+  top: 0;
+  right: 0;
+  margin: 0;
+  padding: 0;
+  font-size: 20px;
```

Comment:
generally I try to keep everything ordered alphabetically and use `em` for `font-size` as a best practice (scales much better on mobile device screen resolutions). you might be able to guess that I hastily copy pasted the `upload-button` and `upload` classes from a blog